engine/api/workflow_run.go

```diff
@@ -313,6 +311,11 @@ func (api *API) stopWorkflowNodeRunHandler() Handler {
 			return sdk.WrapError(errS, "stopWorkflowNodeRunHandler> Unable to stop workflow node run")
 		}
 
+		wr, errLw := workflow.LoadRun(tx, key, name, number)
+		if errLw != nil {
+			return sdk.WrapError(errLw, "stopWorkflowNodeRunHandler> Unable to load workflow run %s", name)
+		}
+
 		if errR := workflow.ResyncWorkflowRunStatus(tx, *wr); errR != nil {
```

Comment:
we should ResyncWorkflowRunStatus the pointer to the workflow run, so it could set the new status the the current workflow run object: 
https://github.com/ovh/cds/blob/refactor_stop_workflow/engine/api/workflow/resync_workflow.go#L59